app/_components/AiMultiModel.jsx

```
@@ -0,0 +1,66 @@
+
+    const onToggleChange=(model,value)=>{
+      setAiModelList((prev)=>
+      prev.map((m)=>
+      m.model===model?{...m, enable: value } :m))
+    }
+
+  return (
+    <div className='flex flex-1 h-[75vh] border-b'>
+        {aiModelList.map((model,index)=>(
+          <div className={`flex flex-col border-r h-full overflow-auto
+          ${model.enable ? 'flex-1 min-w-[400px]' : 'w-[100px] flex-none'}`}>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)